core/runtime/src/main/java/io/quarkus/runtime/util/ManagedScheduledExecutorService.java

```diff
@@ -0,0 +1,50 @@
+package io.quarkus.runtime.util;
+
+import java.util.List;
+import java.util.concurrent.ScheduledExecutorService;
+
+import org.jboss.logging.Logger;
+
+/**
+ * Forwards all method calls to the scheduled executor service returned from the {@link #delegate()} method.
+ * Does not allow shutdown
+ */
+public class ManagedScheduledExecutorService extends ForwardingScheduledExecutorService {
```

Comment:
I don't think that `Managed` is a good name. It's overused and ambiguous. In https://github.com/quarkusio/quarkus/pull/38478 I used the `NoopShutdown` prefix because it's clear what it does. Maybe `Unterminable`?

Reply:
I agree, we can go with `NoopShutdownScheduledExecutorService` ?